recipes/agent_linux.rb

```diff
@@ -13,10 +13,14 @@
   node['newrelic_infra']['agent']['config']['file']
 )
 
+group node['newrelic_infra']['group']['name'] do
+  group_name node['newrelic_infra']['group']['name']
+end
+
 # Setup a service account
-poise_service_user node['newrelic_infra']['user']['name'] do
-  group node['newrelic_infra']['group']['name']
-  only_if { node['newrelic_infra']['features']['manage_service_account'] }
```

Comment:
I guess we are removing this flag, but I don't see it removed:
- neither at [docs](https://github.com/newrelic/infrastructure-agent-chef#attributes)
- nor at [default](https://github.com/newrelic/infrastructure-agent-chef/blob/fd9aa6ffb0c3985451877f941eec7a83d86dad44/attributes/default.rb#L21-L22)

Reply:
I removed flag and updated docs